cloudbuild/nightly/cloudbuild.yaml

```diff
@@ -14,23 +14,78 @@ steps:
       - 'GCS_JAR_LOCATION=${_GCS_JAR_LOCATION}'
       - 'MVN_JAR_LOCATION=${_MVN_JAR_LOCATION}'
 
-# 3. Start the simple table read e2e test.
+# 3. Create the dataproc cluster
   - name: 'gcr.io/$PROJECT_ID/dataproc-flink-bigquery-connector-nightly'
-    id: 'e2e-bounded-read-small-table-test'
+    id: 'create-clusters-bounded-small-table'
+    waitFor: ['init']
+    entrypoint: 'bash'
+    args: ['/workspace/cloudbuild/nightly/nightly.sh', 'create_clusters_bounded_small_table']
+    env:
+      - 'PROJECT_ID=${_PROJECT_ID}'
+      - 'DATAPROC_IMAGE_VERSION=${_DATAPROC_IMAGE_VERSION}'
+      - 'NUM_WORKERS_SMALL_TEST=${_NUM_WORKERS_SMALL_TEST}'
+      - 'CLUSTER_NAME_SMALL_TEST=${_CLUSTER_NAME_SMALL_TEST}'
+      - 'REGION_ARRAY_STRING_SMALL_TEST=${_REGION_ARRAY_STRING_SMALL_TEST}'
+#      - 'TEMP_BUCKET_SMALL_TEST=${_TEMP_BUCKET_SMALL_TEST}'
+#      - 'STAGING_BUCKET_SMALL_TEST=${_STAGING_BUCKET_SMALL_TEST}'
```

Comment:
Why have these comments been added here?

Reply:
ideal to remove them.

Reply:
Done.